docs/content/tutorial/user-management.md

```diff
@@ -88,7 +88,7 @@ This will create a Secret of the pattern `<clusterName>-pguser-postgres` that co
 
 ## Deleting a User
 
-As mentioned earlier, PGO does not let you delete a user automatically: if you remove the user from the spec, it will still exist in your cluster. To remove a user and all of its objects, as a superuser you will need to run [`DROP OWNED`](https://www.postgresql.org/docs/current/sql-drop-owned.html) in each database the user has objects in, and [`DROP ROLE`](https://www.postgresql.org/docs/current/sql-droprole.html)
+As mentioned earlier, PGO does not delete users automatically: after you remove the user from the spec, it will still exist in your cluster. To remove a user and all of its objects, as a superuser you will need to run [`DROP OWNED`](https://www.postgresql.org/docs/current/sql-drop-owned.html) in each database the user has objects in, and [`DROP ROLE`](https://www.postgresql.org/docs/current/sql-droprole.html)
```

Comment:
📝 "As mentioned earlier" … I didn't see this mentioned earlier. Perhaps I missed it or this is leftover from being in another section of the docs. This is a tutorial page; details are in `/architecture/user-management.md`.

Reply:
So it should be safe to ~As mentioned earlier,~

Reply:
As long as you're touching this, want to break it up into <100 char lines? (And I like TJ's change.)

Reply:
I'm generally not in favor of the line length restrictions on the docs. I know this is a debatable issue, but it makes it hard for one who is rapidly adding and editing documentation.

Reply:
I went with @jkatz's suggestion to strike "as mentioned earlier". I'm leaving the line length alone for now, too.